docs/root/api/stats.rst

```diff
@@ -82,3 +82,33 @@ The value of a ``Gauge`` can be incremented, decremented, or reassigned.
 
   // Swift
   gauge.sub(amount: 5)
+
+---------
+``Timer``
+---------
+Use ``Timer`` to track a distribution of time durations.
+You can view the cumulative stats like quantile data (p50/p90/etc.) and average durations.
+
+**Example**::
+
+  // Add a new duration to the underlying timer distribution
+  // Kotlin
+  timer.recordDuration(5)
+
+  // Swift
+  timer.recordDuration(durationMs: 5)
+
+-------------
+``Histogram``
```

Comment:
nit: I'd called this `Distribution`.

Reply:
Sure, I like that.

Reply:
On 2nd thought, I'd like to follow the histogram naming since it's a more common terminology throughout metrics. Do you feel strongly about having this called `Distribution`?